chainerx_cc/chainerx/python/CMakeLists.txt

```diff
@@ -52,3 +52,9 @@ set_target_properties(${module_name}
     PREFIX "${PYTHON_MODULE_PREFIX}"
     SUFFIX "${PYTHON_MODULE_SUFFIX}"
     CXX_VISIBILITY_PRESET "default")
+
+# LAPACK
+if(${LAPACK_FOUND})
+MESSAGE( STATUS "LAPACK is found, defining CHAINERX_ENABLE_LAPACK variable.")
+add_definitions(-DCHAINERX_ENABLE_LAPACK=1)
+endif()
```

Comment:
It looks like you're redefining the variable here as it's already defined here https://github.com/chainer/chainer/pull/7474/files#diff-0a471ff54bba954243aa6dac699759b3R66.

Reply:
Moved `add_definitions(-DCHAINERX_ENABLE_LAPACK=1)` to upper CMakeLists, `chainer/chainerx_cc/CMakeLists.txt`.